pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -956,6 +974,8 @@ func (p *Provider) Create(
 		// Preemptible instances require the following arguments set explicitly
 		args = append(args, "--maintenance-policy", "TERMINATE")
 		args = append(args, "--no-restart-on-failure")
+	} else if providerOpts.useSpot {
+		args = append(args, "--provisioning-model", "SPOT")
```

Comment:
So the user is supposed to use either `--preemptible` or `use-spot`, right? Should we validate that `!providerOpts.useSpot` when `providerOpts.preemptible`?